vagrant/provisioning/roles/microservices/templates/ssl.conf

```diff
@@ -25,11 +25,39 @@ ProxyPassReverse /eureka http://{{ arkcase_host }}:9000/eureka
   </RequireAny>
 </Location>
 
-ProxyPass /kafka http://{{ internal_host }}:9021
-ProxyPassReverse /kafka http://{{ arkcase_host }}:9021
 
 ProxyPass /eureka-dashboard http://{{ arkcase_host }}:9000/eureka-dashboard
 ProxyPassReverse /eureka-dashboard http://{{ arkcase_host }}:9000/eureka-dashboard
+<Location /eureka-dashboard>
+  <RequireAny>
+    Require ip "{{ arkcase_host_address | default('127.0.0.1') }}"
+  </RequireAny>
+</Location>
+
+ProxyPass /control-center http://acm-arkcase:9021
+ProxyPassReverse /control-center http://acm-arkcase:9021
+<Location "/control-center">
+  AddOutputFilterByType INFLATE;SUBSTITUTE;DEFLATE text/html
+  AddOutputFilterByType INFLATE;SUBSTITUTE;DEFLATE application/javascript
+  SubstituteMaxLineLength 10M
+  Substitute "s|/dist/|/control-center/dist/|i"
+  Substitute "s|/2.0/|/control-center/2.0/|i"
+  Substitute "s|/3.0/|/control-center/3.0/|i"
+  <RequireAny>
+    Require ip "{{ arkcase_host_address | default('127.0.0.1') }}"
+  </RequireAny>
+</Location>
+
+ProxyPass /zipkin http://acm-arkcase:9411/zipkin
+ProxyPassReverse /zipkin http://acm-arkcase:9411/zipkin
+<Location "/zipkin">
+  <RequireAny>
+    Require ip "172.28.128.1"
```

Comment:
shall we take 172.28.128.1 from configuration?

Reply:
Missed that one. Thanks.